includes/wikia/services/TemplateClassificationService.class.php

```diff
@@ -40,17 +39,16 @@ class TemplateClassificationService {
 	 * @var array
 	 */
 	static $templateTypes = [
-		self::TEMPLATE_INFOBOX,
-		self::TEMPLATE_QUOTE,
-		self::TEMPLATE_NAVBOX,
-		self::TEMPLATE_FLAG,
-		self::TEMPLATE_CONTEXT_LINK,
-		self::TEMPLATE_REFERENCES,
-		self::TEMPLATE_MEDIA,
 		self::TEMPLATE_DATA,
 		self::TEMPLATE_DESIGN,
+		self::TEMPLATE_FLAG,
+		self::TEMPLATE_INFOBOX,
+		self::TEMPLATE_MEDIA,
 		self::TEMPLATE_NAV,
+		self::TEMPLATE_NAVBOX,
 		self::TEMPLATE_NOT_ART,
+		self::TEMPLATE_QUOTE,
+		self::TEMPLATE_REFERENCES,
 		self::TEMPLATE_UNKNOWN,
```

Comment:
The order of this array is important right now, as it affects the order in the dialog (not ideal, but the way it is right now). So, the previous order with infobox first is correct, see https://github.com/Wikia/app/pull/9065